nextflow_schema.json

```diff
@@ -335,6 +335,20 @@
                     "help_text": "To be used with `--vep_dbnsfp`.",
                     "hidden": true
                 },
+                "dbnsfp_consequence": {
+                    "type": "string",
+                    "fa_icon": "fas fa-database",
+                    "description": "Consequence to annotate with",
```

Comment:
possible values?

Reply:
The parameter "dbnsfp_consequence"  is used to filter/limit the output of the dbnsfp plugin to a specific effect of the variant, i.e. to display only the missense variants (`'consequence=missense_variant'`). The set of consequence terms is defined by the Sequence Ontology and an overview of those used in VEP can be found here: 
`https://www.ensembl.org/info/genome/variation/prediction/predicted_data.html`
If one wants to filter using several consequences, then separate those by using '`&`' (i.e. '`consequence=3_prime_UTR_variant&intron_variant'`.

Reply:
Thanks a lot